xmind/tests/test_sheet_element.py

```diff
@@ -5,7 +5,6 @@
 from xmind.core.const import *
```

Comment:
@mykola-pysarev please be aware that there are no changes in this file except reformatting